opencomputeproject/SAI
# Zitingguo/add lag test

Add lag config and tests.

Config:
- create lag and lag members
- set lag hash and its attribute
- create route and neighbor entry for lag
- remove lag and lag members

Test:
- basic lag config test
- load balance based on src port test
- disable lag member test
- verify lag hash factor do not contain ingress ports test

Signed-off-by: Ziting Guo <zitingguo@microsoft.com>
